handlers.go

```diff
@@ -190,6 +190,11 @@ func (s Server) resourcePatchHandler(w http.ResponseWriter, r *http.Request, id
 		return
 	}
 
+	if resource.Attributes == nil {
+		w.WriteHeader(http.StatusNoContent)
+		return
+	}
```

Comment:
I tried to add some test cases, but it was hard because of the dependence.

A below always returns `Resources`.
https://github.com/elimity-com/scim/blob/1bc6b4a9715467679091130ef48e1051ea1f9c21/resource_handler_test.go#L25-L28

Do you think I have to add some tests?